checkov/terraform/graph_builder/graph_components/module.py

```diff
@@ -3,7 +3,7 @@
 import json
 import os
 from copy import deepcopy
-from typing import List, Dict, Any, Set, Callable, Tuple, TYPE_CHECKING
+from typing import List, Dict, Any, Set, Callable, Tuple, TYPE_CHECKING  # noqa: F401
```

Comment:
out of interest, why is this needed?

Reply:
It raised a mypy failure that was fixed in upstream so this was a patch but actually can be removed